Cargo.toml

```diff
@@ -83,9 +84,9 @@ features = [
 ]
 
 [target.'cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "openbsd", target_os = "netbsd"))'.dependencies]
-wayland-client = { version = "0.29", features = [ "dlopen"], optional = true }
+wayland-client = { version = "0.29", default_features = false,  features = ["use_system_lib"], optional = true }
```

Comment:
Are there any default features to disable in the first place?

Reply:
`dlopen` itself is a default feature.